### PR TITLE
Add navigation event

### PR DESCRIPTION
**Pull request by deepueg**

### CODE REVIEW

**Review by friederbluemle — APPROVED** [review]
👍

**Review by liulianci — APPROVED** [review]

File: android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        return eventType;
+    }
+
+    /**
+    * optional payload associated with the event
+    *
+    * @return String
+    */
+    @NonNull
+    public String getpayload() {
```
Comment (asharpbflat) [inline android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java:81]: Capital `P` for `getPayload()`?

File: android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        }
+    };
+
+    /**
+    * Type of the event. Supported values [BUTTON_CLICK, DID_FOCUS, DID_BLUR]
+    *
+    * @return String
+    */
+    @NonNull
+    public String geteventType() {
```
Comment (asharpbflat) [inline android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java:71]: Capital `E` for `getEventType`?

**Review by asharpbflat — APPROVED** [review]
👍

File: android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java (hunk 2 above)
Comment (deepueg) [reply android/lib/src/main/java/com/ernnavigationApi/ern/api/NavEventData.java:71]: thanks, will fix this in the platform as it is code gen related.

**Pull request merged**